groups/templates/groups/emails/new_comment.txt

```diff
@@ -6,4 +6,4 @@ A new comment has been posted to the discussion "{{ name }}" by {{ creator }}{%
 
 {% else %}.{% endif %}
 
-{% blocktrans with url=comment.get_absolute_url %}Reply to this email to post a response, or visit this URL to reply online: {{ site.domain }}{{ url }}{% endblocktrans %}
+{% blocktrans with url=comment.get_absolute_url domain=site.domain %}Reply to this email to post a response, or visit this URL to reply online: http://{{ domain }}{{ url }}{% endblocktrans %}
```

Comment:
sorry, been distracted by SAT... Where does this `protocol` come from?  Is it just `site.protocol`?